src/App.test.jsx

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+describe('App', () => {
+  it('Should render the header', async () => {
+    render(<App />);
+
+    const heading = await screen.findByText('Home of Futurama!!!');
+    expect(heading).toBeInTheDocument();
```

Comment:
In the future, we can render(< Header /> ) for our component tests without having to use the entire < App /> file. Then, we can place the test file next to our Header component file.